trin-core/src/utp/trin_helpers.rs

```diff
@@ -0,0 +1,107 @@
+// The code here isn't affiliated with stream and isn't part of the uTP implementation
+// this code is used in handling the uTP responses handled in events.rs.
+// just a clarification. It is just for handling the uTP stream higher up what stream releases.
```

Comment:
This line is a little hard to follow. Probably don't need "just a clarification", and I'm not sure what the phrase "higher up what stream releases" is getting at.